matrix-neoboard-widget/src/setupTests.ts

```diff
@@ -28,3 +28,13 @@ import '@testing-library/jest-dom/vitest';
 // definition in jsdom
 import { webcrypto } from 'node:crypto';
 Object.defineProperty(global.globalThis, 'crypto', { value: webcrypto });
+
+// @ts-expect-error This is a polyfill for pdfjs
+window.Promise.withResolvers = function () {
```

Comment:
Why do we need a polyfill here? Support [looks quite good in browsers](https://caniuse.com/?search=withresolvers) . Does our test environment not implement it? If not, is there an issue, that we can link, so that we know when we can remove it?

Reply:
Its not supported by nodejs unless you add a specific node flag (remember our tests do NOT run in any browser env). Its the OLD promise api. 

see https://github.com/mozilla/pdf.js/issues/18006#issuecomment-2078739672

Reply:
![grafik](https://github.com/user-attachments/assets/ad485b7b-eb21-450c-8ec4-fa40fd376ee4)

Node 22 seems to be the first version with support. IMHO this is too new for us.

Reply:
What about adding a comment ~ „remove when Node.JS >= 22“?